src/rules/noUnusedVariableRule.ts

```diff
@@ -72,6 +72,9 @@ export class Rule extends Lint.Rules.TypedRule {
         type: "functionality",
         typescriptOnly: true,
         requiresTypeInfo: true,
+        deprecationMessage: !/^2\.[0-8]\./.test(ts.version)
```

Comment:
you can also use `semver` to test versions

Reply:
yes prefer using `semver`

Reply:
good call, udpated